testing/python/metafunc.py

```diff
@@ -429,7 +429,7 @@ def test_idmaker_autoname(self) -> None:
 
     def test_idmaker_with_bytes_regex(self) -> None:
         result = IdMaker(
-            ("a"), [pytest.param(re.compile(b"foo"), 1.0)], None, None, None, None, None
```

Comment:
Fixed a 'bug' here, right ?

Reply:
Might be nice to add a changelog for it as in "if pytest param contains a wrong number of value there's now going to be a warning" ?

Reply:
IdMaker is not a public API. Normally e.g. in parametrize, this is validated early:

```
x.py::test_it: in "parametrize" the number of names (2):
  ['a', 'b']
must be equal to the number of values (3):
  (1, 2, 3)
```